<?xml version="1.0"?>
<launch> 

  <rosparam file="$(find eurobot)/config/field_params.yaml" command="load" ns="field" />
  <rosparam file="$(find eurobot)/config/particle_filter_params.yaml" command="load" ns="main_robot/particle_filter" />
  <rosparam file="$(find eurobot)/config/particle_filter_params.yaml" command="load" ns="secondary_robot/particle_filter" />
  <rosparam file="$(find eurobot)/config/main_robot_params.yaml" command="load" ns="main_robot" />
  <rosparam file="$(find eurobot)/config/secondary_robot_params.yaml" command="load" ns="secondary_robot" />

</launch>
